docs/eventing/debugging/example.yaml

```diff
@@ -12,32 +12,33 @@ metadata:
 # The event source.
 
 apiVersion: sources.eventing.knative.dev/v1alpha1
-kind: KubernetesEventSource
+kind: ApiServerSource
 metadata:
   name: src
   namespace: knative-debug
 spec:
-  namespace: knative-debug
   serviceAccountName: service-account
+  mode: Resource
+  resources:
+    - apiVersion: v1
+      kind: Event
   sink:
-    apiVersion: eventing.knative.dev/v1alpha1
+    apiVersion: messaging.knative.dev/v1alpha1
     kind: Channel
     name: chan
 
 ---
 
 # The Channel events are sent to.
 
-apiVersion: eventing.knative.dev/v1alpha1
+apiVersion: messaging.knative.dev/v1alpha1
```

Comment:
I think I'd prefer us just creating the InMemoryChannel directly here:
apiVersion: messaging.knative.dev/v1alpha1
kind: InMemoryChannel

and empty spec.